docker/sds-api/spine-directory-service/sds/request/tests/examples/SDS-Endpoint-Example.json

```diff
@@ -0,0 +1,68 @@
+{
+  "resourceType": "Endpoint",
+  "id": "f0f0e921-92ca-4a88-a550-2dbb36f703af",
+  "extension": [
+    {
+      "url": "nhsMHSSyncReplyMode",
+      "valueString": "MSHSignalsOnly"
+    },
+    {
+      "url": "nhsMHSRetryInterval",
+      "valueString": "PT1M"
+    },
+    {
+      "url": "nhsMHSRetries",
+      "valueString": "2"
+    },
+    {
+      "url": "nhsMHSPersistDuration",
+      "valueString": "PT5M"
+    },
+    {
+      "url": "nhsMHSDuplicateElimination",
+      "valueString": "always"
+    },
+    {
+      "url": "nhsMHSAckRequested",
+      "valueString": "always"
+    }
+  ],
+  "identifier": [
+    {
+      "system": "https://fhir.nhs.uk/Id/nhsEndpointServiceId",
+      "value": "urn:nhs:names:services:psis:REPC_IN150016UK0"
+    },
+    {
+      "system": "https://fhir.nhs.uk/Id/nhsMhsFQDN",
```

Comment:
was "system": "https://fhir.nhs.uk/Id/nhsMhsFQD" in the original example.
I think 'N' is missing compared to XML version.